agent/consul/state/catalog_oss_test.go

```diff
@@ -226,7 +226,7 @@ func testIndexerTableGatewayServices() map[string]indexerTestCase {
 		Service: structs.ServiceName{Name: "SerVice"},
 		Port:    50123,
 	}
-	encodedPort := string([]byte{0x96, 0x8f, 0x06, 0, 0, 0, 0, 0, 0, 0})
+	encodedPort := string([]byte{0x80, 0, 0, 0, 0, 0, 0xc3, 0xcb})
```

Comment:
The way integers were encoded was updated in v1.3.3 by https://github.com/hashicorp/go-memdb/pull/114